app/views/admin/auctions/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Create a new auction</h1>
+
```

Comment:
Is there a reason `issue_url` and `github_repo` aren't included in the form? Happy to add them in if we end up wanting those fields.

Also thinking I could have named those a little better. Perhaps `github_repo` -> `repo_url`?